pkgs/by-name/gr/graalvm-ce/package.nix

```diff
@@ -0,0 +1,2 @@
+{ graalvmPackages }:
+graalvmPackages.graalvm-ce
```

Comment:
These are meant to be simple aliases, temporary at that, as discussed. I did it this way so that all-packages.nix doesn't need to be let recursive as that would be a huge change potentially.

Reply:
isn't it already? (basically) all-packages.nix uses `with pkgs;` at the top